packages/core/src/globals/typography.css

```diff
@@ -13,7 +13,7 @@
  * ```
  */
 :root {
-  --font-avenir: 'Avenir Next W01', 'Helvetica Neue', 'Helvetica', 'sans-serif';
+  --font-avenir: 'Avenir Next', 'Helvetica Neue', 'Helvetica', 'sans-serif';
```

Comment:
Where is this font coming from? the font on the site is 'Avenir Next W01', unless I'm missing something?

Reply:
oops, this was a mistake. Will amend now